cmd_test.go

```diff
@@ -26,8 +26,10 @@ var _ = gc.Suite(&CmdSuite{})
 
 func (s *CmdSuite) TestContext(c *gc.C) {
 	ctx := cmdtesting.Context(c)
-	c.Assert(ctx.AbsPath("/foo/bar"), gc.Equals, "/foo/bar")
-	c.Assert(ctx.AbsPath("foo/bar"), gc.Equals, filepath.Join(ctx.Dir, "foo/bar"))
+	c.Check(ctx.AbsPath("/foo/bar"), gc.Equals, "/foo/bar")
+	c.Check(ctx.AbsPath("foo/bar"), gc.Equals, filepath.Join(ctx.Dir, "foo/bar"))
+	homeDir := os.Getenv("HOME")
```

Comment:
I also don't think this is a safe test on Windows, though maybe we're skipping these already? HOME certainly isn't guaranteed to exist there.